### DISCUSSION TITLE
We should direct AMD gpu users to update to the latest 22.7.1 drivers, which includes massive OpenGL performance improvements

### DISCUSSION THREAD

**Discussion by BestUsername1** [ppy/osu#19636] [Ideas] [answered]
I'm not sure if this is the right place to put this, but I use the RX 6600xt graphics card from AMD and previously had major issues with lazer's performance, which I have presumed to be OpenGL related. On AMD's radeon driver 22.6.1 I experienced frequent stutters and relatively low frames (approx 650hz), which persisted even with a full gpu driver reinstall. However, after updating to 22.7.1 lazer can easily run pegged at 1000hz. Sorry that I don't have many technical details to offer, all I can say is that the old driver had red spikes in the "Draw" frametime graph, whereas this driver does not seem to have that at all.

With all that said, I feel we should alert all AMD users of this new driver ASAP as it has hugely changed the way the game looks and feels. Simply put, lazer was very frustrating to play before this driver update, but now it easily rivals stable.

**Comment by peppy** [comment, marked as answer]
I believe most people update their drivers regularly and nothing is required on our end. 650hz is already more than adequate, for what it's worth. you shouldn't need more.

**Reply by BestUsername1** [reply]
I agree that 650hz is more than adequate, I was simply trying to express how large the performance gains were with a simple metric. The more important aspect of this was the stuttering which seems to be entirely gone. I'm not sure if there is an easy way for me to measure this, but I am willing to rollback my drivers and profile the behavior so you can see what I'm talking about.
